compiler/control/OMROptions.cpp

```diff
@@ -3667,10 +3672,13 @@ OMR::Options::jitPostProcess()
    // (~256MB) code caches at startup. The code caches will then have locality which improves performance, but
    // moreover the size of the allocated larger code cache will ensure no trampolines are needed in JIT private
    // linkage.
-   if (self()->getOption(TR_EnableRMODE64))
+   //
+   // Code cache consolidation is enabled on all 64-bit platforms by default. However if RMODE64 is not available
+   // on the OS or if the user specified to disable it we must also disable code cache consolidation.
+   if (!self()->getOption(TR_EnableRMODE64))
 #endif
```

Comment:
This #endif is misplaced. You need to push it down near the next #endif, otherwise codecacheconsolidation is disabled on all platforms except zOS.